src/game/counting.rs

```diff
@@ -0,0 +1,276 @@
+use crate::game::cards::Card;
+use crate::game::scoring::{score_run, Combination, CombinationKind, Score};
+use std::error::Error;
+
+/// Calculates the score during the counting phase where played_cards have already been played and card is now played
+///
+///
+pub fn score_counting_cards_played(
+    played_cards: &[Card],
+    card: Card,
+) -> Result<Score, Box<dyn Error>> {
+    let count: i32 = played_cards.iter().map(|c| c.value).sum::<i32>() + card.value;
+    if count > 31 {
```

Comment:
in the original game, it is an error...the service enforces the rules of the game.  this is the first instance of it, but we should probably put in checks in other places such as correct # cards, no duplicate cards,

Reply:
btw, this particular one is important because the human can do math wrong and try to play the last card and the game "gently" corrects them, whereas all the other checks above only happen with maliciousness.